examples/martini-example/main.go

```diff
@@ -26,7 +26,9 @@ func StartServer() {
 	})
 
 	m.Get("/ping", PingHandler)
-	m.Get("/secured/ping", jwtMiddleware.CheckJWT, SecuredPingHandler)
+	m.Get("/secured/ping", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Hey there! Thanks for submitting a PR.


I believe the root cause of this issue found in the paragraph right above the one linked in the pr https://github.com/go-martini/martini#return-values :

> If a handler returns something, Martini will write the result to the current http.ResponseWriter as a string

The JWT Middleware we provide here will return an error or nil. When you wrap it like you did in this PR, the handler doesnt return anything and doesnt write to the response.

@grounded042 This might be something to think about as we rework the internals.

Aside from that, I think this is a fine workaround, albeit It is not ideal to have to use the middleware like this.

Reply:
Yeah, this is definitely a bummer to have to use this workaround. Let's merge this in for now as it's great to have this, but in the future we should find a better way to handle this.